src/js/common/utils/helpers.js

```diff
@@ -101,14 +101,33 @@ export function dateToMoment(dateField) {
   });
 }
 
+export function dropTimeAndTimezone(date) {
+  let newDate = moment(date).format();
+  const indexOfTime = newDate.indexOf('T');
```

Comment:
This will always resolve to `-1`.  I think you meant to write:
```javascript
const indexOfTime = date.indexOf('T'); // not newDate
```

A side note, I'd recommend renaming `newDate` to `formattedDate`.